<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmDialog</class>
 <widget class="QDialog" name="ConfirmDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>189</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>769</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/menu_icon/icon/dicon.png</normaloff>:/menu_icon/icon/dicon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#ConfirmDialog {
	background: #e1e1e1; 
   border-top: 3px solid #CC181E;
}

QLineEdit {
	border: 1px solid #aaa;
   padding: 4px; 
  color: #333;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget {
	border-bottom: 1px solid #a1a1a1;
 background: white; 
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string>padding: 5px;</string>
            </property>
            <property name="text">
             <string>Url</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string>padding: 5px;</string>
            </property>
            <property name="text">
             <string>Save As</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 7px;</string>
            </property>
            <property name="text">
             <string>File Size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">padding: 7px;</string>
            </property>
            <property name="text">
             <string>Select Category</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="lbl_url">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">padding: 7px 10px; </string>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lbl_size">
            <property name="styleSheet">
             <string notr="true">padding: 3px; </string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="font">
               <font>
                <family>Segoe UI</family>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">
height: 25px; 
width: 100px;</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="lbl_loading">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_start">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 7px 14px; </string>
       </property>
       <property name="text">
        <string>  Start Download  </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">padding: 7px 14px; </string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
